guide/event-driven-and-pooling-consumer-service/tests/event-driven-and-pooling-consumer_test.bal

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/mb;
+import ballerina/runtime;
+import ballerina/task;
+import ballerina/test;
+
+string requestmessage1;
+string requestmessage2;
+string requestmessage3;
+string requestmessage4;
+
+documentation { Define the message queue endpoint for sending geo activities captured from the sensor. }
+endpoint mb:SimpleQueueSender queuesensorGeoMessage {
+    host: "localhost",
+    port: 5672,
+    queueName: "GeoActivities"
+};
+documentation { Define the message queue endpoint to health check messages. }
+endpoint mb:SimpleQueueSender queueHealthCheck {
+    host: "localhost",
+    port: 5672,
+    queueName: "SensorHealth"
+};
+documentation { Define the message queue endpoint for maintenance messages. }
+endpoint mb:SimpleQueueSender queueMaintenance {
+    host: "localhost",
+    port: 5672,
+    queueName: "Maintenance"
+};
+documentation { Define the message queue endpoint for sensor calibration requests. }
+endpoint mb:SimpleQueueSender queueCalibration {
+    host: "localhost",
+    port: 5672,
+    queueName: "Calibration"
+};
+documentation { Attributes associated with the service endpoint. }
+endpoint http:Listener sensorEventListner {
+    port: 8080
+};
+documentation {  via HTTP/1.1. }
+@http:ServiceConfig {
+    basePath: "/"
+}
+service<http:Service> SensorEventService bind sensorEventListner {
+    @http:ResourceConfig {
+        methods: ["POST"], consumes: ["application/json"], produces: ["application/json"],
```

Comment:
shall we format this to break into new lines at commas?

Reply:
Break into new line with commas and set one parameter per line.